### DISCUSSION TITLE
Wechselrichter Type SUN600G3-EU-230

### DISCUSSION THREAD

**Discussion by Caddy-SRX** [tbnobody/OpenDTU#487] [General]
Hi
does openDTU support also the Deye SUNG3-EU-230 or the Bosswerk MI-300 / 600 or is there a plan to do this in near future?

BR

**Comment by tbnobody** [comment]
The SUNG3-EU-230 and Bosswerk already support WiFi out of the box. Therefore no bridge from a vendor specifiy interface like the NRF24 is needed (and thats what openDTU is).

**Reply by Caddy-SRX** [reply]
Hi
I know that this type of inverter have WiFi. But my hope was to integrate the inverter into e.g. Openhab
But thanks for your reply

BR

**Comment by tbnobody** [comment]
Hm not sure about the output of those inverters but are they maybe showing a local served web ui? or is it cloud based only? A local served web page with values (or a API endpoint) could be parsed using e.g. python and published to a mqtt broker. Then the integration in openhab would be straight forwared.

I didn't have a deep look into the project but this sound promising: https://github.com/fr00sch/bosswerk_mi600_solar

**Comment by Caddy-SRX** [comment]
Hi
thanks for the link.
But the result is nearly similar to that what a shelly will also deliver

Your Project is awesome

BR

**Comment by tost11** [comment]
I, was looking for the same because all the features this project already brings with it are nice (display,  mqtt, awesome UI, setting limit).

So if forked the project and added a Deye sun Implementation.

Also, I wanted to get more and often Information out of the Device. First is working: like Pannel Voltage and so on. More often not. It seems the data only changes every 5 minutes.

But however, I managed to get it working. It's not perfect, and not everything is working yet.
But maybe it helps you.

https://github.com/tost11/OpenDTU-Push-Rest-API-and-Deye-Sun/tree/feature/deye-sun
